dockers/docker-fpm-frr/staticd.static_route.conf.j2

```diff
@@ -0,0 +1,52 @@
+{% block static_route %}
+{% if STATIC_ROUTE %}
+{% set glb_rt = {} %}
+{% set vrf_rt = {} %}
+{% for rt_key in STATIC_ROUTE %}
+{%   if rt_key is not string() %}
+{%     if vrf_rt.has_key(rt_key[0]) or vrf_rt.update({rt_key[0]:{}}) %}
+{%     endif %}
```

Comment:
how to understand `if` without anything in the body?

Reply:
That is a wrong configuration. The attribute 'nexthop' is mandatory. If empty it will get exception when generating .conf file.